modules/serverless-hotfix-patch.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * install/serverless-upgrades.adoc
+
+:_content-type: CONCEPT
+[id="serverless-hotfix-patch_{context}"]
+= Serverless operator maintenance release upgrades
+
+== Latest and maintenance releases
+
+Beginning with {ServerlessProductName} 1.29, the different product versions are available as follows:
+
+* The latest release is available through the `stable` channel.
+* The maintenance release is available through its version-based channel, for example, `stable-1.29`. The maintenance release is the release prior to the latest release, for example if `stable` is 1.30, then maintenance release is `stable-1.29`.
+
+Using a version-based channel allows you to stay within a specific `x.y` release. Additionally, it prevents an upgrade to the latest version of the product, which might contain breaking changes.
+
+To switch to the maintenance release, update the channel parameter in the subscription object YAML file from `stable` to the corresponding version-based channel, such as `stable-1.29`.
+
+== Patches and hotfixes for maintenance releases
+
+Same as the stable release, the maintenance release is subject to patches and hotfixes, which help to maintain your deployment up-to-date with critical bug and security fixes.
+
+* Patches are updates that are distributed as z-releases, for example, {ServerlessProductName} 1.29.1 is the patch that provides updates made since version 1.29.0.
+* Hotfixes are fixes that require zero downtime and are used directly in production. They are different from the usual updates in that they upgrade the customer's deployed version, and not the latest released version.
++
+Hotfixes might not be available to all customers immediately. However, changes introduced by hotfixes often become available to all customers as part of a future release.
++
+When a hotfix pertaining to your deployment is available, you will be given the hotfix CatalogSource to update your subscription and obtain the hotfix.
```

Comment:
Is there a link about this for the customer to go from here?

Reply:
@skonto According to the hotfix process document, specific customers are contacted by the productization team specifically:
"So after the build and distribution (errata publication) its on Engineering/Software Production to tell CEE or the requester of the hotfix, what the CatalogSource image is so that they can communicate to the customer/administrator how to consume the hotfix."